website/versioned_docs/version-v1.3_alpha/integrating.md

```diff
@@ -91,6 +158,20 @@ maci-cli verify \
     --tally-file tally.json # this file is generated in genProofs
 ```
 
+### Finalize in `maci/contracts`
+
+```
+pnpm merge:localhost --poll 0
+pnpm run prove:localhost --poll 0 \
+    --coordinator-private-key "macisk.1751146b59d32e3c0d7426de411218172428263f93b2fc4d981c036047a4d8c0" \
+    --process-zkey ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test.0.zkey \
+    --tally-zkey ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test.0.zkey \
+    --tally-file ../cli/tally.json \
+    --output-dir ../cli/proofs/ \
+    --tally-wasm ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test_js/TallyVotes_10-1-2_test.wasm \
+    --process-wasm ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test_js/ProcessMessages_10-2-1-2_test.wasm
+```
+
 When poll finishes, [Tally contract](https://github.com/privacy-scaling-explorations/maci/blob/dev/contracts/contracts/Tally.sol) emits the event with poll address so you can track the state changes.
```

Comment:
nit: "When a poll finishes" 
"The [Tally contract]()"
"emits an event" 
"with the poll address"